cdap-standalone/bin/cdap.bat

```diff
@@ -24,7 +24,7 @@ SET CDAP_HOME=%CDAP_HOME:~0,-5%
 SET JAVACMD=%JAVA_HOME%\bin\java.exe
 
 REM Specifies Web App Path
-SET WEB_APP_PATH=%CDAP_HOME%\web-app\local\server\main.js
+SET UI_PATH=%CDAP_HOME%\ui\alpha\server.js

```

Comment:
why is the path still alpha? If the old UI is gone, then this should be just /ui/